package.json

```diff
@@ -15,8 +15,8 @@
     "scripts": {
         "create:migration": "cd packages/shared/lib/db && knex migrate:make $1 --esm --knexfile ./knexfile.cjs",
         "undo:migration": "cd packages/shared/lib/db && knex migrate:rollback --esm --knexfile ./knexfile.cjs",
-        "prettier-format": "prettier --config .prettierrc \"./**/*.ts\" --write",
-        "prettier-watch": "onchange './**/*.ts' -- prettier --write {{changed}}",
+        "prettier-format": "prettier --config .prettierrc \"./**/*.{ts,tsx}\" --write",
+        "prettier-watch": "onchange './**/*.{ts,tsx}' -- prettier --write {{changed}}",
```

Comment:
Are you using those commands yourself? because prettier is already handled by eslint

Reply:
I ran `prettier-format` manually to get these updates because my local wasn't formatting previously on watch.